starter-html-example/autorun.brs

```diff
@@ -0,0 +1,44 @@
+function main()
+	mp = CreateObject("roMessagePort")
+
+	' Create HTML Widget
+	widget = CreateHTMLWidget(mp)
+	widget.Show()
+
+	'Event Loop
+	while true
+		msg = wait(0,mp)
+		print "msg received - type=";type(msg)
+		if type(msg) = "roHtmlWidgetEvent" then
+			print "msg: ";msg
+		end if
+	end while
+end function
+
+function CreateHTMLWidget(mp as object) as object
+
+	' Enable Web Inspector
+	reg = CreateObject("roRegistrySection","html")
+	reg.Write("enable_web_inspector","1")
+	reg.Flush()
```

Comment:
I don't think this is necessary anymore and imo it should be removed b/c this is a simple example